deploy/webhook.yaml

```diff
@@ -0,0 +1,265 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: service-apis
+  labels:
+    name: service-apis
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: service-apis-admission
+webhooks:
+  - name: validate.networking.x-k8s.io
+    matchPolicy: Equivalent
+    rules:
+      - operations: [ "CREATE" ]
+        apiGroups: [ "networking.x-k8s.io" ]
+        apiVersions: [ "v1alpha1" ]
+        resources: [ "httproutes" ]
```

Comment:
Should we instead include all resources in here?
The reason being - as new versions are released, validations for other resources will be added. Manifests like these are sticky and proliferate quick. A version bump is a simple enough change to pull off, but I've experienced users not updating all the manifests (unless using Helm) and run into weird corner cases.

Reply:
I'm uncertain what other resources we should should add at this time. I do plan to make this a helm based deploy once I have all of the other webhook functions complete.

Reply:
We have to be somewhat careful here as we don't want to start off with more permission than we actually need. Maybe we'll end up having webhook validation in place for most resources before many people install the webhook. I'd lean towards starting with a minimal list here but attempting to expand quickly as we add new validations (and likely before many users have installed this).